packages/apollo-engine-reporting/src/treeBuilder.ts

```diff
@@ -1,10 +1,5 @@
-import {
-  GraphQLResolveInfo,
-  GraphQLError,
-  ResponsePath,
-  responsePathAsArray,
-} from 'graphql';
-import { Trace, google } from 'apollo-engine-reporting-protobuf';
+import { GraphQLError, GraphQLResolveInfo, ResponsePath } from 'graphql';
+import { google, Trace } from 'apollo-engine-reporting-protobuf';
```

Comment:
Strongly prefer keeping these no-op, one-off changes out of change sets, but not asking you to revert it.